app/helpers/admin/flexible_page_helper.rb

```diff
@@ -0,0 +1,28 @@
+module Admin::FlexiblePageHelper
+  def render_flexible_page_content_fields(schema, edition, path = "", output_html = "")
+    schema["properties"].each do |key, property|
+      next_path = path.empty? ? key : "#{path}.#{key}"
+      case property["type"]
+      when "string"
+        output_html += render "govuk_publishing_components/components/input", {
```

Comment:
I'd move these `render` calls out into their own (private?) methods, in anticipation of this helper growing.